src/useLocalStorage.ts

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
```

Comment:
```suggestion
import * as React from "react";
```

Reply:
Added allowSyntheticDefaultImports": true,, see above

Reply:
I'll make it * as React, and then React.functions